packages/cli/src/commands/verify/spec.ts

```diff
@@ -0,0 +1,129 @@
+import { ParamDetails, Arg, Option } from '../../register-command';
+
+import { TxParams } from '@openzeppelin/upgrades';
+import NetworkFile from '../../models/files/NetworkFile';
+import { DEFAULT_TX_TIMEOUT } from '../../models/network/defaults';
+
+import ProjectFile from '../../models/files/ProjectFile';
```

Comment:
The spec file is always loaded even if the `verify` command wasn't the one invoked. To avoid loading unnecessary files why we should do dynamic `await import`s inside the `async` functions below.

Reply:
This is used to create a new `ProjectFile`, which is in turn used to provide the defaults for the `optimizer` and `optimizer-runs` settings. I placed it here since this is the only common scope to both of their `details` functions.

We could refactor this into importing the file in both `details`, creating the new `ProjectFile` in each if needed, or putting this into a function that memoizes these values. WDYT?

Reply:
I think it's okay to import in both and create it in both separately.